### DISCUSSION TITLE
Can't connect to AWS DocumentDB

### DISCUSSION THREAD

**Discussion by amjad-chalktalk** [getredash/redash#6820] [Q&A]
We setup a Amazon document db with mongo version 5,when trying to connect it to redash we are getting this error

**`Schema refresh error`**
we are running redash as container as a part of docker compose service which uses [this docker image](https://hub.docker.com/r/redash/redash/tags?page=1&name=10.1.0.b50633)
Any ideas about this issue?
Note:- we were able to connect our application code to the db successfully (just to make sure that it's Redash issue not DB issue)

**Comment by yswtrue** [comment]
same here
